doc/source/serve/production.md

```diff
@@ -547,7 +556,12 @@ $ python
 
 The price has updated! The same request now returns `10` instead of `6`, reflecting the new price.
 
-You can update any setting in any deployment in the config file similarly. You can also add new deployment settings or remove old deployment settings from the config. This is because `serve deploy` is **idempotent**. Your Serve application's will match the one specified in the latest config you deployed– regardless of what config files you deployed before that.
+### Code Updates
+
```

Comment:
Can we add a note somewhere that changing the top-level `import_path` or `runtime_env` will tear down all the deployments and restart them?